doc/changelog.md

```diff
@@ -254,171 +193,201 @@
   array(4.6)
   ```
 
+  Note that the current implementation of reverse mode AD is restricted to 1st order derivatives,
+  but you can still use `catalyst.grad(method="fd")` is still available to perform a finite
+  differences approximation of _any_ differentiable function.
+
 * Add support for the new PennyLane arithmetic operators.
-  [#250](https://github.com/PennyLaneAI/catalyst/pull/250)
+  [(#250)](https://github.com/PennyLaneAI/catalyst/pull/250)
 
-  PennyLane is in the process of replacing ``Hamiltonian`` and ``Tensor`` observables with a set of
+  PennyLane is in the process of replacing `Hamiltonian` and `Tensor` observables with a set of
   general arithmetic operators. These consist of
   [Prod](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.Prod.html),
   [Sum](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.Sum.html) and
   [SProd](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.SProd.html).

```

Comment:
@josh146 should this be an "improvement" since the impact seems fairly low for a new feature?